drivers/ieee802154/ieee802154_dw1000.c

```diff
@@ -1309,6 +1320,20 @@ static int dwt_configure_rf_phy(struct dwt_context *ctx)
 	/* Set IEEE 802.15.4 compliant mode */
 	sys_cfg &= ~DWT_SYS_CFG_PHR_MODE_11;
 
+	if (rf_cfg->phr_mode_ext) {
+		/* Long frame mode */
+		sys_cfg |= DWT_SYS_CFG_PHR_MODE_11;
+	}
+
+
+	/* Disable TX smart power by default */
+	sys_cfg |= DWT_SYS_CFG_DIS_STXP;
+
+	if (rf_cfg->smart_power_en && rf_cfg->dr == DWT_BR_6M8) {
+		/* Enable smart TX power control for 6.8Mbps */
+		sys_cfg &= ~DWT_SYS_CFG_DIS_STXP;
```

Comment:
This is default behavior according to the DW1000 Manual, "When DIS_STXP is 0 and the data rate is configured to 6.8 Mbps, Smart Tx power is enabled.".  This change seems to be redundant.